frontend/src/interfaces/user.ts

```diff
@@ -1,7 +1,8 @@
 export default interface User {
+  id?: number;
```

Comment:
why optional si id and password?

Reply:
Optional ang id sir kay inig register, dili man maka-add ug id since auto increment man sa database.
For password, ang response to fetch users kay dili man mag-contain sa password sir.